pyproject.toml

```diff
@@ -9,9 +9,9 @@ authors = ["Ülgen Sarıkavak <ulgensrkvk@gmail.com>"]
 ipdb = "0.13.13"
 ipython = "8.21.0"
 # Formatters
-django-upgrade = "1.15.0"
+django-upgrade = "1.16.0"
```

Comment:
If you have `ruff --select=UP` then why bother with `pyupgrade`?

Reply:
@cclauss I'm not sure what you are referring. There is no `pyupgrade` in this file.